### ISSUE TITLE
make PythonAPI ARGS="--python-version=3.7"

### ISSUE THREAD

**Issue by yihsinma** [kochlisGit/autonomous-vehicles-agent#6]
Hello!
I download your "code" directory and paste it into "Carla/PythonAPI/".
need to make PythonAPI ARGS="--python-version=3.7" again？

**Comment by kochlisGit** [comment]
Which carla version are you using?

**Comment by yihsinma** [comment]
0.9.12

**Comment by yihsinma** [comment]
> Which carla version are you using?

0.9.12

**Comment by kochlisGit** [comment]
Yes, this vesion requires python 3.7 to be installed. Then pip (of python 3.7) install carla==0.9.12
